back/spec/acceptance/baskets_spec.rb

```diff
@@ -32,10 +32,7 @@
 
         expect(json_response.dig(:data, :id)).to eq @basket.id
         expect(json_response.dig(:data, :type)).to eq 'basket'
-        expect(json_response.dig(:data, :attributes)).to include(
-          total_budget: 2250,
-          budget_exceeds_limit?: false
-        )
+        # expect(json_response.dig(:data, :attributes, :total_votes)).to eq 5 # TODO: use other voting method to guarantee 5
```

Comment:
Why can't these tests be fixed now if we're actually testing a continuous_budgeting_project here? then add additional tests for other voting types?

Reply:
Budgeting contexts overwrite the votes, so the 5 is no longer 5 for budgeting.